packages/hardhat-etherscan/src/solc/bytecode.ts

```diff
@@ -308,35 +300,27 @@ export async function compile(
 
   await taskRun(TASK_COMPILE);
 
-  const buildInfoFiles = await artifacts.getBuildInfoFiles();
+  const contractBuildInfos: ContractBuildInfo[] = [];
```

Comment:
I adapted this plugin more than need to have some first-hand experience with the API.